alma8/config.yaml

```diff
@@ -11,6 +11,7 @@ variables:
   CMS_COMPATIBLE_OS: el8
   DEFAULT_PACKAGES: coreutils-single
   EPEL_PACKAGES:
+  DOCKER_NAME: alma8
```

Comment:
by the way, no need to add `DOCKER_NAME` , this is already set by default for each container build job